Resources/public/js/views/fields/ez-keyword-view.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-keyword-view', function (Y) {
+    "use strict";
+    /**
+     * Provides the Keyword field view
+     *
+     * @module ez-url-view
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The keyword field view
+     *
+     * @namespace eZ
+     * @class KeywordView
+     * @constructor
+     * @extends eZ.FieldView
+     */
+    Y.eZ.KeywordView = Y.Base.create('keywordView', Y.eZ.FieldView, [], {
+        /**
+         * Check if the array of the field value is empty
+         *
+         * @method _isFieldEmpty
+         * @protected
+         * @return {Boolean}
+         */
+        _isFieldEmpty: function () {
+
+            if (this._getFieldValue().length) {
+                return false;
+            }
+            return true;
+        },
+
+
+        /**
+         * Returns the value to be used in the template. If the value is not
+         * filled, it returns undefined otherwise an object with a `link` and a
+         * `text` entries.
+         *
+         * @method _getFieldValue
+         * @protected
+         * @return Object
+         */
+        _getFieldValue: function () {
```

Comment:
the method in the parent class does exactly the same thing, so you don't need to redefine this method.